apf/apfAggregateNumbering.h

```diff
@@ -0,0 +1,37 @@
+#ifndef APF_AGGREGATE_NUMBERING_H_
+#define APF_AGGREGATE_NUMBERING_H_
+#include "apfNew.h"
+#include <PCU.h>
+namespace apf
+{
+  class Field;
+  class FieldShape;
+  class Sharing;
+  class Mesh;
+  class MeshEntity;
+  template <class T>
+  class AggregateNumberingOf;
+  typedef AggregateNumberingOf<int> AggNumbering;
+  typedef AggregateNumberingOf<long> GlobalAggNumbering;
+  AggNumbering * createAggNumbering(Field * f,
```

Comment:
This should probably just return Numbering* since AggNumbering inherits from Numbering and the entire point of the class is to be used as a Numbering